src/dq_suite/output_transformations.py

```diff
@@ -656,9 +761,8 @@ def get_highest_severity_from_validation_result(
     severity_priority = {"fatal": 3, "error": 2, "warning": 1, "ok": 0}
 
     for result in validation_result.get("results", []):
-        if result.get("success") is False:
-            expectation_type = result["expectation_config"]["type"]
-            rule_name = humps.pascalize(expectation_type)
+        if not result.get("success"):
```

Comment:
Nice change to use of `if not`! :+1:

Same question here about use of `humps.pascalize`.